Practise11_05/Task2/Gallery.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practise11_05.Task2
+{
```

Comment:
The gallery should be responsible for all worker and plant interaction.
Also gallery should aggregate worker and plants.